cid_test.go

```diff
@@ -401,6 +465,25 @@ func TestPrefixRoundtrip(t *testing.T) {
 	}
 }
 
+func TestBadPrefixFromBytes(t *testing.T) {
+	_, err := PrefixFromBytes([]byte{0x80})
+	if err == nil {
+		t.Fatal("expected error from PrefixFromBytes for bad byte 0")
+	}
+	_, err = PrefixFromBytes([]byte{0x01, 0x80})
+	if err == nil {
+		t.Fatal("expected error from PrefixFromBytes for bad byte 1")
+	}
+	_, err = PrefixFromBytes([]byte{0x01, 0x01, 0x80})
+	if err == nil {
+		t.Fatal("expected error from PrefixFromBytes for bad byte 2")
+	}
+	_, err = PrefixFromBytes([]byte{0x01, 0x01, 0x01, 0x80})
+	if err == nil {
+		t.Fatal("expected error from PrefixFromBytes for bad byte 3")
```

Comment:
to save you a bit of work, you don't need to repeat PrefixFromBytes in these error messages, because they'll be preceded by the test name anyway, TestBadPrefixFromBytes.